app/config/default.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.13.0-rc.1",
+  "version": "1.13.3",
```

Comment:
Should this release not be `1.13.1` instead? we seem to be jumping 3 patch versions here